discord/commands/core.py

```diff
@@ -363,7 +363,7 @@ def get_cooldown_retry_after(self, ctx: ApplicationContext) -> float:
             If this is ``0.0`` then the command isn't on cooldown.
         """
         if self._buckets.valid:
-            bucket = self._buckets.get_bucket(ctx)
+            bucket = self._buckets.get_bucket(ctx)  # type: ignore
```

Comment:
Why are types ignored?

Reply:
`cooldowns.CooldownMapping.get_bucket`, and whole class accept Message as args

Reply:
Altering the type of get_bucket might be more appropriate

Reply:
> Altering the type of get_bucket might be more appropriate

![image](https://github.com/Pycord-Development/pycord/assets/61660055/faa65c60-ea13-4dc0-aff1-5116ac0e9384)
hm it was ignored previously also

Reply:
and i think its out of scope also as it would be better to rename "message" arg also